libs/utils/error.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import "errors"
+
+// ErrorContains reports whether any error in err's tree matches any error in targets tree.
+func ErrorContains(err, target error) bool {
```

Comment:
Hmmm, error.Is does the same though. It recursively unwraps error in error tree until it finds the target. What is this helper for?